build/windows/installer/conf/fluent-cri-parser.conf

```diff
@@ -0,0 +1,6 @@
+<parse>
+  @type  regexp
+  expression ^(?<time>[^ ]+) (?<stream>stdout|stderr) (?<logtag>[^ ]*) (?<log>.*)$
```

Comment:
> Is the parser for linux different?

Parser logic is same for Linux and windows but for linux for we use Fluentbit and for windows we use Fluentd. hence we need to maintain separate parsers since Fluentd and Fluentbit has separate syntax for parsers.